storage/pebble/events_hash.go

```diff
@@ -0,0 +1,65 @@
+package pebble
```

Comment:
we need a way to rollback the start height when starting up with `--force-start-height`

Reply:
On a side note, we also need to fix the `--force-start-height`, as the EVM state index is not yet idempotent. Meaning that we'll still encounter some state mismatch issues, this time from the ingestion part itself.